web_poet/serialization/api.py

```diff
@@ -110,42 +112,59 @@ def deserialize_leaf(cls: Type[T], data: SerializedLeafData) -> T:
     return f_ser.f_deserialize(cls, data)  # type: ignore[attr-defined]
 
 
-def _get_fqname(cls: type) -> str:
-    """Return the fully qualified name for a type.
+def _get_name_for_class(cls: type) -> str:
```

Comment:
The coverage warnings here are weird. This line should be executed on import time, it can't be the case that the module is never imported.

Reply:
After some change (I would say it's tox 4 but I don't think it would matter) the coverage output indeed changed from "all is covered except several lines in 3 files" to this, where all defs and imports etc. are not covered.

Reply:
Interesting! If that's tox issue, probably it has something to do with their new default to use isolated builds.